src/RpcClient/RpcClient.cs

```diff
@@ -435,7 +451,7 @@ public async Task<string> GetNewAddressAsync()
         public async Task<BigDecimal> GetWalletBalanceAsync(string assetId)
         {
             byte decimals = await new Nep5API(this).DecimalsAsync(UInt160.Parse(assetId.AsScriptHash())).ConfigureAwait(false);
-            var result = await RpcSendAsync("getwalletbalance", assetId).ConfigureAwait(false);
+            var result = await RpcSendAsync(GetRpcName(), assetId).ConfigureAwait(false);
```

Comment:
There can not get the correct method name, if use `GetRpcName()` and not in the first line of method.🤣